packages/flutter_tools/bin/fuchsia_builder.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:args/args.dart';
+
+import '../lib/src/base/context.dart';
+import '../lib/src/base/logger.dart';
+import '../lib/src/flx.dart';
+import '../lib/src/globals.dart';
+
+const String _kOptionSnapshotter = 'snapshotter-path';
+const String _kOptionTarget = 'target';
+const String _kOptionPackages = 'packages';
+const String _kOptionOutput = 'output-file';
+const String _kOptionSnapshot = 'snapshot';
+const String _kOptionDepfile = 'depfile';
+const String _kOptionWorking = 'working-dir';
+const List<String> _kOptions = const <String>[
+  _kOptionSnapshotter,
+  _kOptionTarget,
+  _kOptionPackages,
+  _kOptionOutput,
+  _kOptionSnapshot,
+  _kOptionDepfile,
+  _kOptionWorking,
+];
+
+Future<Null> main(List<String> args) async {
```

Comment:
Maybe this should return `Future<int>` and we should change the calls to exit to just return the value.